src/main/java/com/minecolonies/coremod/util/SoundUtils.java

```diff
@@ -162,9 +162,9 @@ public static void playInteractionSoundAtCitizenWithChance(@NotNull final World
                 prefix = citizen.getWorkBuilding().getJobName();
             }
 
-            if ("GuardTower".equals(prefix) && citizen.getWorkBuilding() instanceof AbstractBuildingGuards)
+            if ("GuardTower".equals(prefix) && citizen.getWorkBuilding() instanceof AbstractBuildingGuardsNew)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'GUARD_TOWER' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)